wnfs/src/private/node.rs

```diff
@@ -714,6 +710,13 @@ mod tests {
 
     use super::*;
 
+    #[async_std::test]
+    async fn generate_double_random_is_ne() {
+        let rng = &mut TestRng::deterministic_rng(RngAlgorithm::ChaCha);
+        let (first, second) = PrivateNode::generate_double_random(rng);
+        assert_ne!(first, second);
+    }
```

Comment:
Thanks for the refactor. 
Thinking this can work as a non-async function.

```suggestion
    #[test]
    fn generate_double_random_is_ne() {
        let rng = &mut TestRng::deterministic_rng(RngAlgorithm::ChaCha);
        let (first, second) = PrivateNode::generate_double_random(rng);
        assert_ne!(first, second);
    }
```

Reply:
Thanks - Re-wired the calls directly to util as it was single-use fn